canonical-web-and-design/jaas-dashboard
# Fix prettier regressions

## Done

Fix prettier regressions

## QA

- Pull code
- Run `npx prettier --check 'src/**/*'`
- Verify no errors
